cluster-service/deploy/helm/values.yaml

```diff
@@ -0,0 +1,265 @@
+
+# Memory request.
+memoryRequest: "150Mi"
+
+# Memory limit.
+memoryLimit: "1Gi"
+
+# CPU request.
+cpuRequest: "50m"
+
+# CPU limit.
+cpuLimit: "1"
+
+# Client ID for the clusters-service manage identity.
+azureCsMiClientId: ""
+
+# Azure region the cluster-service instance is running in.
+region: "westus3"
+
+# Base64 encoded service principal credentials for CS. This is temporary for P1.
+csServicePrincipalCredsBase64: ""
+
+# The client id of the first party application identity.
+azureFirstPartyApplicationClientId: ""
+
+# The name of the Key Vault containing keys, secrets and certificates used to support services running on the service cluster.
+serviceKeyVaultName: ""
+
+# The Tenant ID of service cluster.
+tenantId: ""
+
+# The name of the secret that contains the first party application certificate bundle.
+fpaCertName: ""
+
+# If set to true the connection to the database is performed without TLS.
+databaseDisableTLS: "false"
+
+# Authentication method to use when connecting to the database. Accepted values are 'az-entra', 'postgres'
+databaseAuthMethod: "az-entra"
+
+# URL of the storage account blob service, e.g. https://<storage-account>.blob.core.windows.net/
+oidcIssuerBlobServiceUrl: ""
+
+# OIDC base issuer URL, e.g. https://<storage-account>.z1.web.core.windows.net/
+oidcIssuerBaseUrl: ""
+
+# The client id of the service principal that represents the ARM Helper Identity.
+azureArmHelperIdentityClientId: ""
+
+# The name of the secret that contains the ARM Helper Indentity certificate bundle.
+azureArmHelperIdentityCertName: ""
+
+# The principal id of the service principal that represents the mock first party application identity.
+azureArmHelperMockFpaPrincipalId: ""
+
+# The name of the secret that contains the mock managed identities certificate bundle.
+azureMiMockServicePrincipalCertName: ""
+
+# The client id of the mock managed identities service principal.
+azureMiMockServicePrincipalClientId: ""
+
+# The principal id of the mock managed identities service principal.
+azureMiMockServicePrincipalPrincipalId: ""
+
+# Base64 encoded provision shards config.
+provisionShardsConfig: ""
+
+# ocm client id
+clientId: "foo"
+
+# ocm secret
+clientSecret: "bar"
+
+# If true an in cluster database will be deployed.
+deployLocalDatabase: true
+
+# The name of the K8s secret where CS DB connection information is placed
+databaseK8sSecretName: "ocm-cs-db"
+
+# The hostname of the postgres server/service. It can be a K8s service name
+databaseHost: "ocm-cs-db"
+
+# Username for PostgreSQL user that will be used for accessing the database.
+databaseUser: "ocm"
+
+# Password for the PostgreSQL connection user.
+databasePassword: "TheBlurstOfTimes"
+
+# Name of the PostgreSQL database accessed.
+databaseName: "ocm-cs-db"
+
+# Host port
+databasePort: "5432"
+
+# The name of the managed identities data plane audience resource.
+managedIdentitiesDataPlaneAudienceResource: "https://dummy.org"
```

Comment:
One constraint we've to take onto account regarding this field's value is that we cannot check it in code for higher envs (stage/prod). So when we get to those envs, please make sure that this is not checked in. @tony-schndr

Reply:
What should the values for those environments be?

Reply:
We don't know, but whatever the value of them, they shouldn't be checked in any repo.

Reply:
To be specific, 
The dev and int envs value can be any dummy https url; this can be checked in the repo.
For stage and prod envs, these will be real value of the msi dataplane. They can't be checked in any repo.